tests/helper_functions.py

```diff
@@ -33,6 +33,9 @@
 AWS_METADATA_IP = "169.254.169.254"  # Used to fetch AWS Instance and User metadata.
 LOCALHOST = "127.0.0.1"
 PROTOBUF_REQUIREMENT = "protobuf<4.0.0"
+# pydantic 2.6.3 has breaking changes that affect the mlflow server
+# TODO: remove this after https://github.com/SeldonIO/MLServer/pull/1595 is applied
+PYDANTIC_REQUIREMENT = "pydantic<2.6.3"
```

Comment:
+1

Reply:
is it possible to import this from `mlflow.models.container` instead? in case we forget to update one